### ISSUE TITLE
windows

### ISSUE THREAD

**Issue by SabraHashemi** [zcsd/Vision_Inspection#1]
can i use this repo for windows? does it have any constraint for windows run?

**Comment by zcsd** [comment]
yes, you can. But for .pro file,  it will be different library path.

**Comment by ZiyaElri** [comment]
Hi, firstly thank you for project. It's really useful project and very good source. But ı have a questions. 
1-) I am using ubuntu 23.04 and ı followed your steps correctly but when ı build the project, ı took a warning like " :-1: error: cannot find -lpython3.6m: No such file or directory". I ınstalled python3.6 and neccessary depencies but ı cant solve this. How can ı solve this problem? Can you give me an advice? 
2-)Can ı use this project with basler dart camera (3.0 usb camera) ?

Thank you for everything. Care yourself :) Good luck, good work.
